keywords/utils.py

```diff
@@ -279,14 +279,17 @@ def add_new_fields_to_doc(doc_body):
     return doc_body
 
 
-def compare_sg_cbl_docs(cbl_db, db, sg_docs):
+def compare_docs(cbl_db, db, docs_dict):
```

Comment:
this method name change will affect the usage in test_delta_sync.py. you may want to update the method name in that file also

Reply:
good point, updated it